spring/src/main/java/org/axonframework/spring/authorization/UnauthorizedMessageException.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2010-2024. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.spring.authorization;
+
+import org.axonframework.common.AxonNonTransientException;
+
+/**
+ * Exception indicating that a message has been rejected due to a lack of authorization.
+ *
+ * @author Roald Bankras
+ *
+ * @since 4.10.0
```

Comment:
Nit: I know I nudged you to add `4.10.0`, but it should be `4.11.0`....which is my bad, sorry for that.